features/openhab-addons/src/main/feature/feature.xml

```diff
@@ -175,6 +175,13 @@
         <bundle start-level="80">mvn:org.openhab.binding/org.openhab.binding.exec/${project.version}</bundle>
     </feature>
 
+    <feature name="openhab-bindign-fatekplc" description="Fatek PLC Binding" version="${project.version}">
+        <feature>openhab-runtime-base</feature>
+        <feature>openhab-runtime-compat1x</feature>
```

Comment:
Not yet.

`SerialFatekConnectionFactory` is only skeleton, which must be implemented and then depend on `openhab-transport-serial` should be add.